examples/hello_world/lib/main.dart

```diff
@@ -2,15 +2,28 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
```

Comment:
I assume this is your local test harness. I would recommend taking this and adding it as a unit test in the packages/flutter/test/widgets area. Please make sure to revert the changes here either way.

Reply:
Yeah I just hijacked this as a testing playground-I'll revert the changes and create unit tests. I mostly wanted some feedback before building testing to make sure I'm on the right track.